script.js

```diff
@@ -0,0 +1,64 @@
+const form = document.getElementById('form');
+const inputBook = document.getElementById('book');
+const inputAuthor = document.getElementById('author');
+const Container = document.getElementById('container');
+
+// Add Books
+
+const addBook = () => {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const authorbook = inputAuthor.value;
+    const titlebook = inputBook.value;
+    const itemlist = document.createElement('div');
+    itemlist.className = 'Itemlist';
+
+    const obj = {
+      Title: titlebook,
+      Author: authorbook,
+    };
+
+    const book = `
+    <ul class=''>
+    <li>${obj.Title}</li>
+    <li>${obj.Author}</li>
+    </ul>
+    `;
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'remove';
+    removeBtn.id = 'removeBtn';
+    removeBtn.innerText = 'Remove';
+
+    itemlist.innerHTML = book;
+    itemlist.appendChild(removeBtn);
+    Container.append(itemlist);
+
+    const result = JSON.stringify(obj);
+    localStorage.setItem('data', result);
+  });
+};
+
+addBook();
+
+const getDataStored = JSON.parse(localStorage.getItem('data'));
```

Comment:
- [OPTIONAL] Currently when the user adds a book, the book is not added to a list in the local storage, and after the addition of another book, the previous one is lost and when the user reloads the page there is no list of books being read. It would be great if the book is added to a collection and this collection is saved to local storage, and after the reload the app loads all the books from that collection and renders in the page like the project required.